napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1008,6 +1008,70 @@ def cli(self, commands):
             cli_output[py23_compat.text_type(command)] = output
         return cli_output
 
+    def get_environment(self):
+        """
+        Get environment facts.
+
+        power and fan are currently not implemented
+        cpu is using 1-minute average
+        """
+
+        environment = {}
+        # sys_resources contains cpu and mem output
+        sys_resources = self._send_command('show system resources')
+        temp_cmd = 'show env temp'
+
+        # cpu
+        environment.setdefault('cpu', {})
+        environment['cpu'][0] = {}
+        environment['cpu'][0]['%usage'] = 0.0
+        system_resources_cpu = napalm.base.helpers.textfsm_extractor(self, 'system_resources',
+                                                                     sys_resources)
+        from decimal import Decimal
```

Comment:
@ktbyers no problem I went ahead and rebased. Let me know if there is a better way here to subtract the decimal idle value from 100 to get the cpu usage.